tools/lint_requirements.py

```diff
@@ -15,23 +15,34 @@ def main(argv: Sequence[str] | None = None) -> int:
     parser = argparse.ArgumentParser()
```

Comment:
I would just exclude the overrides file from this linter -- it doesn't apply at all